src/modules/scraper/adapter/messaging/TokenDetailsConsumer.ts

```diff
@@ -25,6 +27,9 @@ export class TokenDetailsConsumer {
     const token = await this.ethProvidersService.getCachedToken(sourceChainId, tokenAddr);
     if (!token) throw new Error("Token not found");
     await this.depositRepository.update({ id: deposit.id }, { tokenId: token.id });
+    await this.scraperQueuesService.publishMessage<TokenPriceQueueMessage>(ScraperQueue.TokenPrice, {
+      depositId,
+    });
```

Comment:
after the token details were fetched, proceed to get the token price by publishing a new queue message